pyemma/coordinates/data/featurization/misc.py

```diff
@@ -73,6 +72,7 @@ class CustomFeature(Feature):
     >>> data = reader.get_output()
 
     """
+    _serialize_version = 0
```

Comment:
this in general cannot be pickled and should be handled differently (raise? warn?)